### ISSUE TITLE
Reduce logger call statements and enhance the content of the log messages

### ISSUE THREAD

**Issue by ogajduse** [SatelliteQE/robottelo#12431]
There are basically 4 logger calls with no intermediate steps.
I would either reduce the number of calls and call the logger only once or place the logging statements in the correct place.

Also, if the log mentions a content view, it is a good practice to mention the ID or name of the object (CV in this case) that the log message is referring to.

_Originally posted by @ogajduse in https://github.com/SatelliteQE/robottelo/pull/11553#discussion_r1311307592_

**Comment by vijaysawant** [comment]
Reduced logger and merged into one, requested changes can be seen here https://github.com/SatelliteQE/robottelo/pull/11553/files#diff-22533d183099fdbb3c3070a86cc26b50b384780f0544d6b2d8c1e53e077a1b5aR253
